Benchmark-Models/Blasi_CellSystems2016/petab_select/output/petab/measurements.tsv

```diff
@@ -1,16 +1,253 @@
+observable_x_0ac	condition	inf	0.722193168
+observable_x_0ac	condition	inf	0.7346053206
+observable_x_0ac	condition	inf	0.7317104869
+observable_x_0ac	condition	inf	0.7368055433
+observable_x_0ac	condition	inf	0.7353015316
+observable_x_0ac	condition	inf	0.7860370077
 observable_x_0ac	condition	inf	0.7730402558
+observable_x_4ac	condition	inf	0.0025586787
+observable_x_4ac	condition	inf	0.0025276712
+observable_x_4ac	condition	inf	0.0032089475
+observable_x_4ac	condition	inf	0.0026968148
+observable_x_4ac	condition	inf	0.0035432949
+observable_x_4ac	condition	inf	0.0032944225
+observable_x_4ac	condition	inf	0.0032371439
+observable_x_4ac	condition	inf	0.0038015215
+observable_x_4ac	condition	inf	0.0068750527
+observable_x_4ac	condition	inf	0.0056489969
+observable_x_4ac	condition	inf	0.0049831642
+observable_x_4ac	condition	inf	0.0033456395
+observable_x_4ac	condition	inf	0.0035895127
+observable_x_4ac	condition	inf	0.0027663549
+observable_x_4ac	condition	inf	0.0030647938
+observable_x_4ac	condition	inf	0.0031698565
+observable_x_4ac	condition	inf	0.0017445951
+observable_x_4ac	condition	inf	0.0014080101
+observable_x_k05	condition	inf	0.0357434956
+observable_x_k05	condition	inf	0.0299751007
+observable_x_k05	condition	inf	0.0319485374
+observable_x_k05	condition	inf	0.0356067978
+observable_x_k05	condition	inf	0.0358038875
+observable_x_k05	condition	inf	0.0338619565
+observable_x_k05	condition	inf	0.032286999
+observable_x_k05	condition	inf	0.0386392741
+observable_x_k05	condition	inf	0.0298921207
+observable_x_k05	condition	inf	0.0344702628
+observable_x_k05	condition	inf	0.035647086
+observable_x_k05	condition	inf	0.0416663478
+observable_x_k05	condition	inf	0.0364716656
+observable_x_k05	condition	inf	0.0357468678
+observable_x_k05	condition	inf	0.037757738
+observable_x_k05	condition	inf	0.035665225
+observable_x_k05	condition	inf	0.0328347091
 observable_x_k05	condition	inf	0.0301433562
-observable_x_k08	condition	inf	0.0306241172
-observable_x_k12	condition	inf	0.0623801676
-observable_x_k16	condition	inf	0.0341717472
+observable_x_k05k08	condition	inf	0.0056806576
+observable_x_k05k08	condition	inf	0.0051190482
+observable_x_k05k08	condition	inf	0.0064222373
+observable_x_k05k08	condition	inf	0.006415159
+observable_x_k05k08	condition	inf	0.0071723039
+observable_x_k05k08	condition	inf	0.0053909205
+observable_x_k05k08	condition	inf	0.0070625017
+observable_x_k05k08	condition	inf	0.0055944146
+observable_x_k05k08	condition	inf	0.0062533439
+observable_x_k05k08	condition	inf	0.0060135768
+observable_x_k05k08	condition	inf	0.0073291191
+observable_x_k05k08	condition	inf	0.0063897149
+observable_x_k05k08	condition	inf	0.0065985939
+observable_x_k05k08	condition	inf	0.0066172451
+observable_x_k05k08	condition	inf	0.0063324372
+observable_x_k05k08	condition	inf	0.006899493
+observable_x_k05k08	condition	inf	0.0038748389
 observable_x_k05k08	condition	inf	0.0037448182
-observable_x_k05k12	condition	inf	0.0388013075
```

Comment:
@dilpath why some of the measurements are removed? could you clarify?

Reply:
They are not removed, just appear at different positions. The old measurements were accidentally the "last" measurement for each observable in the new measurements table.

The new measurements table now has the measurements that were in the original problem.

Reply:
e.g.

https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/pull/158/files#diff-499b12588c267d39114860aa60bfaef8c97d003e857d6c0b5dd6539c8d667b03L16

https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/pull/158/files#diff-499b12588c267d39114860aa60bfaef8c97d003e857d6c0b5dd6539c8d667b03R37